waves-ext/src/main/resources/application.conf

```diff
@@ -11,6 +11,11 @@ waves {
         # Extension's port
         port = 6887
       }
+
+      order-script-validation {
+        # Accounts in base58 that are allowed accessing blockchain state
```

Comment:
Let's write "Public keys of accounts" instead of just "Accounts"

Reply:
fxd